packages/web3/std/nft_collection_interface.ral

```diff
@@ -1,8 +1,5 @@
 Interface INFTCollection {
-   event Minted(minter: Address, tokenIndex: U256, tokenId: ByteVec)
-
-   pub fn getName() -> ByteVec
-   pub fn getSymbol() -> ByteVec
+   pub fn getCollectionUri() -> ByteVec
```

Comment:
Let's add a standard for the collection metadata in the comment?

Reply:
Done :+1: